src/ui/screens/MyIdentityScreen/ManageIdentityScreen.test.tsx

```diff
@@ -0,0 +1,74 @@
+import {NavigationProp, RouteProp} from '@react-navigation/native';
+import {AccountsStackParamList} from '@ui/navigation/navigation';
+import React from 'react';
+import {render, waitFor, fireEvent} from 'src/testUtils';
+import {ManageIdentityScreen} from './ManageIdentityScreen';
+import {manageIdentityScreen} from '@ui/navigation/routeKeys';
+import {Alert, Linking} from 'react-native';
+
+const navigation = {
+  navigate: () => jest.fn(),
+  setOptions: () => jest.fn,
+} as unknown as NavigationProp<AccountsStackParamList>;
+
+const route = {
+  params: {
+    address: '14yx4vPAACZRhoDQm1dyvXD3QdRQyCRRCe5tj1zPomhhS29a',
+  },
+} as RouteProp<AccountsStackParamList, typeof manageIdentityScreen>;
+
+const openURLSpy = jest.spyOn(Linking, 'openURL');
+
+describe('ManageIdentityScreen', () => {
+  it('should render the loading view when data is fetching', async () => {
+    const {findByText, findAllByText} = render(<ManageIdentityScreen navigation={navigation} route={route} />);
+    await findByText('Address');
+    await findAllByText('Email');
+    await findByText('info@purestake.com');
+    await findAllByText('Twitter');
+    await findByText('@nachortti');
+    await findAllByText('Legal');
+    await findByText('Raul Romanutti');
+    await findAllByText('Riot');
+    await findByText('@raul.rtti:matrix.parity.io');
+    await findAllByText('Web');
+    await findByText('www.nachortti.com');
+    await findByText('Update Identity');
+    await findByText('Request Judgement');
+    await findByText('Set Sub-identities');
+    await findByText('Clear Identity');
+    await findByText('Polkascan');
+    await findByText('View externally');
+  });
+
+  const identities = [
+    ['@nachortti', 'https://twitter.com/@nachortti'],
+    ['@raul.rtti:matrix.parity.io', 'https://matrix.to/#/@raul.rtti:matrix.parity.io'],
+    ['www.nachortti.com', 'www.nachortti.com'],
+  ];
+
+  it.each(identities)('should click on %s to navigate to %s url', async (id, url) => {
```

Comment:
`it('should linkify linkable identities' ...`

Let's include all the identities and assert as well that when pressed, `openUrl` should not be called